src/Umbraco.Web.UI.Client/src/packages/data-type/search/data-type-search.server.data-source.ts

```diff
@@ -39,7 +39,7 @@ export class UmbDataTypeSearchServerDataSource implements UmbSearchDataSource<Um
 		if (data) {
 			const mappedItems: Array<UmbDataTypeSearchItemModel> = data.items.map((item) => {
 				return {
-					href: '/section/settings/workspace/data-type/edit/' + item.id,
+					href: '/umbraco/section/settings/workspace/data-type/edit/' + item.id,
```

Comment:
This should use the <base> tag, which it will if it is relative instead (without a leading slash):


```suggestion
					href: 'section/settings/workspace/data-type/edit/' + item.id,
```

Reply:
Now outdated.